docs/package.json

```diff
@@ -14,9 +14,9 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.18",
+    "@docusaurus/core": "^2.1.0",
```

Comment:
@mikicho  This is a major change, could you check if /docs 'npm run build' and 'npm start' works?

We might break our website

Reply:
check